src/CartingService/DataAccess/Entities/Cart.cs

```diff
@@ -0,0 +1,59 @@
+using CartingService.DataAccess.Interfaces;
+using CartingService.DataAccess.ValueObjects;
+using MongoDB.Driver;
+
+namespace CartingService.DataAccess.Etities;
+
+public class Cart : ICartEntity
+{
+    public Guid Guid { get; set; }
```

Comment:
format "{tick}_{guid}"

Reply:
used [ObjectId](https://www.mongodb.com/docs/manual/reference/method/ObjectId/) instead of formatted string as ObjectId contains:
- A 4-byte timestamp, representing the ObjectId's creation, measured in seconds since the Unix epoch.
- A 5-byte random value generated once per process. This random value is unique to the machine and process.
- A 3-byte incrementing counter, initialized to a random value.